src/ReactiveUI.Tests/TestUtilsTest.cs

```diff
@@ -7,10 +7,5 @@ namespace ReactiveUI.Tests
 {
     public class TestUtilsTest
     {
-        [Fact]
-        public async Task WithAsyncScheduler()
```

Comment:
This test just seemed to break everything... I'm not even really sure what it's testing, so thought losing it seemed safest...